### PR TITLE
Add brief 3rd-party review for urlbox.io

### PR DESCRIPTION
**Pull request by nicl**
(We want to use this to help with the DCR migration.)

### CODE REVIEW

**Review by DavidLawes — APPROVED** [review]
ahh, sorry if I should have added this in myself!

File: docs/architecture/3rd party technical review/004-urlbox.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+The contract renews each month and cancellation is easy via their website.
+
+## Data privacy engineering review
+
+There are no known data privacy concerns. urlbox.io will be used to capture
+screenshots of publicly accessible pages on www.theguardian.com.
+
+## Security engineering review
+
+urlbox.io is an (HTTP) API service, accessed directly without a specific
```
Comment (paulbrown1982) [inline docs/architecture/3rd party technical review/004-urlbox.md:33]: Where will you be storing the API access key?

File: docs/architecture/3rd party technical review/004-urlbox.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+client-library. There are no known security concerns.
```
Comment (paulbrown1982) [inline docs/architecture/3rd party technical review/004-urlbox.md:34]: How will we manage credentials to the website login page?

Comment (nicl) [reply docs/architecture/3rd party technical review/004-urlbox.md:34]: @paulbrown1982 I'd be open to recommendations here. Typically, these have been shared and developers maintain them separately - e.g. through a password manager/browser password storage.

File: docs/architecture/3rd party technical review/004-urlbox.md (hunk 1 above)
Comment (nicl) [reply docs/architecture/3rd party technical review/004-urlbox.md:33]: On local machines only.

Comment (paulbrown1982) [reply docs/architecture/3rd party technical review/004-urlbox.md:33]: Could it be stored in AWS Parameter Store and downloaded to the local machine via the AWS CLI?

File: docs/architecture/3rd party technical review/004-urlbox.md (hunk 2 above)
Comment (paulbrown1982) [reply docs/architecture/3rd party technical review/004-urlbox.md:34]: That's fine, maybe we can say that we will always maintain 2 copies on developer machines, plus the cardholder?

**Review by paulbrown1982 — APPROVED** [review]
👍

**Pull request merged**